### PR TITLE
New blog post: Getting to Know Matt Davis

### PR DESCRIPTION
**Pull request by amyrbrown**
Again, not tested locally because I can't get the website compiling on my computer.

### CODE REVIEW

File: blog/2015/03/getting-to-know-matt-davis.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+(So lots of orchestrating data flowing through statistical models.)
+At previous jobs I've worked on web applications, astronomy data pipelines
+and observation planning tools, and on data analysis/visualization
+in atmospheric science.
+</p>
+
+<h3>What programming languages and other technology do you regularly use?</h3>
+<p>
+I use the scientific Python stack
+(numpy, scipy, Pandas, matplotlib) for almost everything,
```
Comment (r-gaia-cs) [inline blog/2015/03/getting-to-know-matt-davis.html]: @amyrbrown Could you replace "numpy, scipy" with "NumPy, SciPy"? I'm very picky about software names. =)

**Comment by gvwilson** [comment]
@jiffyclub @amyrbrown is this one ready to merge?

**Comment by amyrbrown** [comment]
As far as I'm concerned they're both ready to go, apart from the dates are wrong. Can you fix them or should I update the PRs? 

The plan is to post one of these every couple of weeks, so don't merge both. (From now on I think we will stage them on the communications repo and only issue a PR against site when it's time to post them. @adina does that sound good?)

**Comment by howe-lab** [comment]
Sounds fantastic, thanks!

**Comment by adina** [comment]
Sounds great, thanks!

**Comment by gvwilson** [comment]
OK, I'll merge one tomorrow (and fix the date) then do the second next week.

**Comment by amyrbrown** [comment]
Splendid, thank you!

**Comment by r-gaia-cs** [comment]
@gvwilson Can you remove the

> This is the second in our "Getting to Know" series of contributor profiles.

This is now the first one. =)

**Comment by gvwilson** [comment]
Thanks - pushing now.

On 2015-04-27 7:29 AM, Raniere Silva wrote:
>
> @gvwilson <https://github.com/gvwilson> Can you remove the
>
>     This is the second in our "Getting to Know" series of contributor
>     profiles.
>
> This is now the first one. =)
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/site/pull/907#issuecomment-96615435>.
>

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org

**Pull request merged**